lib/mumuki/laboratory/status/submission/submission.rb

```diff
@@ -0,0 +1,43 @@
+module Mumuki::Laboratory::Status::Submission
+  include Mumuki::Laboratory::Status
+
+  require_relative './unknown'
```

Comment:
I always put the requires out of the module. What are the benefits of doing this way?

Reply:
I needed them to be after the module definition and before the STATUSES array, that's the only reason :stuck_out_tongue: